LinkedListImplementation/LinkedListImplementation.cs

```diff
@@ -0,0 +1,105 @@
+﻿using LinkedListImplementation.Interfaces;
+﻿using System.Reflection.Metadata.Ecma335;
+
+namespace LinkedListImplementation
+{
+    public class MyLinkedList<T> : ILinkedList<T> where T : IComparable<T>
+    {
+        public class Node<T>
+        {
+            public T Data { get; private set; }
```

Comment:
Better use read-only property for Data

Reply:
Is it because we get access in constructor and after that we dont need to change Data?

Reply:
Right